src/Routing/Adapter/Laravel.php

```diff
@@ -5,8 +5,8 @@
 use Illuminate\Http\Request;
 use Illuminate\Routing\Route;
 use Illuminate\Routing\Router;
-use Illuminate\Routing\RouteCollection;
 use Dingo\Api\Contract\Routing\Adapter;
+use Illuminate\Routing\RouteCollection;
```

Comment:
No, it's not. It actually make no sense for me, but Style CI complains about it https://styleci.io/analyses/qoQRrp